### PR TITLE
Inbox ID credential proto

### PR DESCRIPTION
**Pull request by 37ng**

### CODE REVIEW

File: proto/mls/message_contents/credential.proto [hunk 1]
```diff
@@ -8,15 +8,6 @@ import "mls/message_contents/association.proto";
 option go_package = "github.com/xmtp/proto/v3/go/mls/message_contents";
 option java_package = "org.xmtp.proto.mls.message.contents";
 
-// A credential that can be used in MLS leaf nodes
```
Comment (neekolas) [inline proto/mls/message_contents/credential.proto:11]: If we're putting this into main, let's just leave this old credential as-is and add the new one. That way it's not a breaking change

Comment (37ng) [reply proto/mls/message_contents/credential.proto:11]: I rebased this to `identity` branch, deleted part is for CI to pass.

File: proto/identity/credential.proto [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// Credentials
+syntax = "proto3";
+
+package xmtp.identity;
+
+option go_package = "github.com/xmtp/proto/v3/go/mls/message_contents";
```
Comment (neekolas) [inline proto/identity/credential.proto:6]: The `go_package` should be `github.com/xmtp/proto/v3/go/identity`

File: proto/identity/credential.proto [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+option java_package = "org.xmtp.proto.mls.message.contents";
```
Comment (neekolas) [inline proto/identity/credential.proto:7]: This should be `org.xmtp.proto.identity`

File: proto/mls/message_contents/credential.proto (hunk 1 above)
Comment (neekolas) [reply proto/mls/message_contents/credential.proto:11]: I still think we might want to keep the old one around for a bit. Once we start using this in `libxmtp` we are going to have to update every single reference to the old credential immediately or compiles will fail.

**Review by neekolas — APPROVED** [review]

**Pull request merged**